canon/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## [Unreleased]
 
+## [0.6.4] 2021-06-04
+
+### Changed
+- Change `Id::encoded_len` to a correct implementation
+
+### Removed
+- Remove `Repr`
```

Comment:
Not sure what this mean. `Repr` still exists in this PR…?

Reply:
Changed to deprecated